editor/modes/visual-editor/index.js

```diff
@@ -2,12 +2,14 @@
  * External dependencies
  */
 import { connect } from 'react-redux';
+import { first, last } from 'lodash';
 
 /**
  * WordPress dependencies
  */
 import { __ } from 'i18n';
 import { Component, findDOMNode } from 'element';
+import { LETTER_A, SMALL_LETTER_A } from 'utils/keycodes';
```

Comment:
Two things:

- Maybe `LOWER_LETTER_A` (lowercase, uppercase distinction)
- Do we need to capture uppercase A? Cmd-Shift-A is not typically a valid combination for Select All behavior.